substrate/runtime-io/without_std.rs

```diff
@@ -27,13 +27,15 @@ use rstd::intrinsics;
 use rstd::vec::Vec;
 pub use rstd::{mem, slice};
 
-#[lang = "panic_fmt"]
+#[panic_implementation]
 #[no_mangle]
-pub extern fn panic_fmt(_fmt: ::core::fmt::Arguments, _file: &'static str, _line: u32, _col: u32) {
+pub fn rust_begin_panic(info: &::core::panic::PanicInfo) -> ! {
```

Comment:
slightly bikeshedding (so won't block approval) but i prefer simple names if possible - this could be simply `panic` like before. also, anything with `begin` in its names instinctively makes me look for something with similar with `end` in its name.